Sources/TokamakCore/Fiber/Element.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  File.swift
+//
+//
+//  Created by Carson Katri on 2/15/22.
+//
+
+/// A reference type that points to a `Renderer`-specific element that has been mounted.
+/// For instance, a DOM node in the `DOMFiberRenderer`.
+public protocol Element: AnyObject {
+  associatedtype Data: ElementData
```

Comment:
Similar concern about `Data` name, maybe `Content` could work better? Besides, there's an established precedent for using `Content` name in SwiftUI as a generic parameter.